app/models/user.rb

```diff
@@ -116,4 +119,8 @@ def self.from_oauth_hash(auth_hash)
   def role_on(proposal)
     RolePicker.new(self, proposal)
   end
+
+  def requires_profile_attention?
```

Comment:
It's used only on view logic now, but imo it represents business logic that could be used anywhere. E.g. writing a rask task to report on or notify all users who require some attention to their profile.